src/devices/Board/tests/RpiBoardDtOverlayTests.cs

```diff
@@ -30,78 +33,78 @@ public void DisplayActivations()
             try
             {
                 // This is not really a test but a way to see the configuration of the Raspberry PR used to run all the tests
-                Console.WriteLine("This is not really a test but a way to see the configuration of the Raspberry PR used to run all the tests.");
-                var isI2c = board.IsI2cActivated();
-                Console.WriteLine($"Is I2C overlay actvated? {isI2c}");
+                _output.WriteLine("This is not really a test but a way to see the configuration of the Raspberry PR used to run all the tests.");
+                var isI2c = _board.IsI2cActivated();
+                _output.WriteLine($"Is I2C overlay actvated? {isI2c}");
 
                 for (int busid = 0; busid < 2; busid++)
```

Comment:
Are all of the PIs in our CI PI 3's? Just asking, because RPI4 has up to 6 I2C busses (and Uarts, and SPIs)

Reply:
AFAIK yes they are all Pi3s